pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -4,15 +4,15 @@
 , openssh, pcre2, bash
 , asciidoc, texinfo, xmlto, docbook2x, docbook_xsl, docbook_xml_dtd_45
 , libxslt, tcl, tk, makeWrapper, libiconv
-, svnSupport, subversionClient, perlLibs, smtpPerlLibs
+, svnSupport ? false, subversionClient, perlLibs, smtpPerlLibs
```

Comment:
Btw: If we add the defaults here we should probably also remove them from `git` in `pkgs/top-level/all-packages.nix`:
```
    svnSupport = false;         # for git-svn support
    guiSupport = false;         # requires tcl/tk
    sendEmailSupport = false;   # requires plenty of perl libraries
```

And IMO this should go into a dedicated commit as it's unrelated.

Reply:
Moving that to another PR.